### PR TITLE
Closes Virtual Env Issue

### PR DESCRIPTION
**Pull request by Umekko**
### Describe your change:

[TASK] Виртуальное окружение

closes https://github.com/SENATOROVAI/docs/issues/30

Missed 4th question, answered in comments for the issue 

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Add or change doctests? -- Note: Please avoid changing both code and tests in a single pull request.
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/SENATOROVAI/pre-commit/blob/main/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

File: Python/venv.ipynb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{
```
Comment (Umekko) [inline Python/venv.ipynb]: Missed answer for 4-th question:
Will answer here 
pip install package name
conda install package name

File: .mergify.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+pull_request_rules:
```
Comment (ruslansenatorov) [inline .mergify.yml:1]: test

**Review by ruslansenatorov — COMMENTED** [review]
tst

File: HomeWork/new_stash.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
+# ---
```
Comment (ruslansenatorov) [inline HomeWork/new_stash.py:1]: twst1111

**Review by ruslansenatorov — COMMENTED** [review]
JJJJJ

File: Python/venv.ipynb [hunk 4]
```diff
@@ -0,0 +1,927 @@
...
+   "metadata": {},
+   "source": [
+    "![image.png](attachment:image.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "####  4. Как установить необходимые пакеты внутрь виртуального окружения для conda/venv?\n",
```
[the remark this answers is not in the document]
Comment (Umekko) [reply Python/venv.ipynb:72]: pip install package name
conda install package name

**Review by ruslansenatorov — COMMENTED** [review]
ревью

File: .github/workflows/isort.yml [hunk 5]
```diff
@@ -35,6 +36,6 @@ jobs:
         git config --global user.name "GitHub Actions"
         git config --global user.email "actions@github.com"
```
Comment (ruslansenatorov) [inline .github/workflows/isort.yml:37]: 1

File: .gitignore [hunk 6]
```diff
@@ -237,99 +237,6 @@ target/
...
-
-# Adobe Files
-*.psd  # Adobe Photoshop
-*.ai   # Adobe Illustrator
-*.indd # Adobe InDesign
-*.xd   # Adobe XD
-
-
+.pre-commit-config.yaml
+DIRECTORY.md
```
Comment (ruslansenatorov) [inline .gitignore:241]: 2

File: Python/made-easy/chapter_1_intro_to_ds.ipynb [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+   "name": "python3"
+  },
+  "language_info": {
+   "name": "python",
+   "version": "3.10.9"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Comment (ruslansenatorov) [inline Python/made-easy/chapter_1_intro_to_ds.ipynb:300]: test

File: Math/calculus/lektorium/Lectorium_part_1_22_10_2024.pdf
Comment (ruslansenatorov) [inline Math/calculus/lektorium/Lectorium_part_1_22_10_2024.pdf:1]: проверь файл

**Pull request closed without merging**